### PR TITLE
feat(VList): update to md2

### LINKED ISSUE #3597: [Bug Report] List-tiles used as activator doesn't get list__tile--link class
### Versions and Environment
**Vuetify:** 1.0.8
**Vue:** 2.5.16
**Browsers:** Firefox 60.0
**OS:** Windows 10

### Steps to reproduce
 * Use a list-tile as an activator for a dialog

### Expected Behavior
I'd expect that the list tile has the have hover effects as when an @click property is provided - pointer mouse cursor, shaded tile.

### Actual Behavior
No indicator is given that the tile is clickable.

### Reproduction Link
[https://codepen.io/mjwalds/pen/NYRBNR/](https://codepen.io/mjwalds/pen/NYRBNR/)

### Other comments
There are a few easy workarounds - 
 * empty @click property (`<v-list-tile slot="activator" @click="">`) - inherently messy and an obvious workaround
 * Use a variable for the dialog rather than an activator - In our particular case, this ends up adding a lot of code and variables we don't need.

### LINKED ISSUE #3834: [Bug Report] group list collapse animation breaks when group list item uses ripple and closes the group list
### Versions and Environment
**Vuetify:** 1.0.14
**Vue:** 2.5.16
**Browsers:** Chrome 65.0.3325.181
**OS:** Mac OS 10.13.4

### Steps to reproduce
Create a list group with sub items. Each sub item should have the ripple directive. When clicking on a sub item, the item will call a function that will ultimately close the group list. 

The function that closes this group list will delay changing the value (open close state) of the v-group-list component. This delay can be only a few milliseconds. In the example it's 200 ms.

Click on list item in the group list and observe the behavior.

### Expected Behavior
When a list group closes, there should be a closing animation for the group list.

### Actual Behavior
The group list closes instantly, with no animation.

### Reproduction Link
[https://codepen.io/MatthewAry/pen/ZxPdbp?editors=1010](https://codepen.io/MatthewAry/pen/ZxPdbp?editors=1010)

### Other comments
So far this issue has only been observed on MacOS. But I was able to open a VM running Windows 10 and reproduce the issue in Microsoft Edge. This issue was also reproduced in Firefox 59.0.2 on a Mac.

### LINKED ISSUE #4180: [Feature Request] v-list-group keyboard accessibility
### Problem to solve
The current v-list-group component cannot be accessed via tabbing on a keyboard.

### Proposed solution
Adding a tabindex and a keydown event to the genGroup method will allow keyboard access.

### LINKED ISSUE #5713: [Bug Report] v-menu with attach prop inside a v-list-group causes left padding
### Versions and Environment
**Vuetify:** 1.3.9
**Vue:** 2.5.17
**Browsers:** Firefox 63.0
**OS:** Mac OS 10.13

### Steps to reproduce
After expanding the list component, and opening the v-select that has an attach prop, all menu items in the v-select have left padding applied

### Expected Behavior
The menu items should be aligned to the left

### Actual Behavior
The menu items all have the same left-padding applied

### Reproduction Link
<a href="https://codepen.io/anon/pen/eQMWPE?editors=1010" target="_blank">https://codepen.io/anon/pen/eQMWPE?editors=1010</a>

### Other comments
I've tried playing around with putting the v-select into a v-list-tile, but that causes the v-select to be hidden.

### LINKED ISSUE #5949: [Bug Report] v-list three line tile - subtitle not clamping vertically at 2 lines and then ellipsis
### Versions and Environment
**Vuetify:** 1.3.14
**Vue:** 2.5.21
**Browsers:** Chrome 71.0.3578.98
**OS:** Windows 10

### Steps to reproduce
open codepen, then set viewport to mobile breakpoint size.

### Expected Behavior
subtitle within tile to clamping vertically at 2 lines and then ellipsis when there is more text than will fit in the tile

### Actual Behavior
subtitle text not ellipsing, instead the text spills out into the tile above

### Reproduction Link
<a href="https://codepen.io/pbyford/pen/pqRrJJ?editors=1010" target="_blank">https://codepen.io/pbyford/pen/pqRrJJ?editors=1010</a>

### LINKED ISSUE #6199: [Feature request] v-icon in v-list should be inherit
I'm working with main nav in Dashboard. I wanna the main-nav like image. And I set active-class and class to set default color for both text and active-class. The text ok, but the v-icon is not. To set color to. I need to set more class for icon.

Main-Nav Design
image

This is my Main-Nav
```vue
      <v-list class="main-nav">
        <v-list-tile
          v-for="item in items"
          :key="item.text"
          :to="item.url"
          active-class="light-blue lighten-5 primary--text"
          class="blue-grey--text"
        >
          <v-list-tile-action>
            <v-icon>{{ item.icon }}</v-icon>
          </v-list-tile-action>
          <v-list-tile-content>
            <v-list-tile-title class>{{ item.text }}</v-list-tile-title>
          </v-list-tile-content>
        </v-list-tile>
      </v-list>
```
To apply for v-con, I need apply this style
```
.main-nav .v-icon {
  color: inherit;
}
```

I think it would be convenient for us.

### LINKED ISSUE #6266: [Bug Report] Nested Lists in Nav Drawer Caret/Arrow Glitch (Mobile)
### Versions and Environment
**Vuetify:** 1.4.3
**Vue:** 2.5.22
**Browsers:** Safari, Google Chrome
**OS:** iOS

### Steps to reproduce
1. Create a nested list within a nav drawer
2. In order to see the issue, the nested list needs to go from less than 100% height before opening to more than 100% after it has been opened. (I think)
3. Open and close the nested list; the caret will disappear/become displaced upon closing.
4. Only happens on mobile. (iOS for sure, unsure about Android)

### Expected Behavior
Smooth caret animation

### Actual Behavior
Brief displacement/disappearing of arrow

### Reproduction Link
<a href="https://codepen.io/DabbyDabber/pen/VgemPJ" target="_blank">https://codepen.io/DabbyDabber/pen/VgemPJ</a>

### Other comments
I tried removing the scrollbar with css, but it did not work.

### LINKED ISSUE #6455: [Bug Report] v-list-tile component renders anchors with no href rather than buttons
### Versions and Environment
**Vuetify:** 1.4.8
**Vue:** 2.6.5
**Browsers:** Chrome 72.0.3626.81
**OS:** Mac OS 10.14.3

### Steps to reproduce
Create a `v-list` and `v-list-tile` list without `href`, `to` or `nuxt` props but with an `v-on:click=" "` event.

### Expected Behavior
The list items should render as `<button>` elements.

### Actual Behavior
They render as `<a>` (anchor) elements with no href. This is incorrect because they do not navigate anywhere.

### Reproduction Link
<a href="https://codepen.io/anon/pen/MLGYjr?&editable=true&editors=101" target="_blank">https://codepen.io/anon/pen/MLGYjr?&editable=true&editors=101</a>

### PR DESCRIPTION
**Pull request by johnleider**
## Description
Update list to md2 spec

fixes #3597
fixes #3834
fixes #4180
fixes #5713
fixes #5949
fixes #6199
fixes #6266
fixes #6455

## Motivation and Context
Converting framework to md2

## How Has This Been Tested?
jest

## Markup:

<details>

```vue
<template>
  <v-layout row>
    <div class="ma-5" style="max-width: 400px;">
      <v-list dense class="mb-5 py-0">
        <v-list-item color="white" three-line>
          <v-list-item-avatar color="grey"></v-list-item-avatar>
          <v-list-item-content>
            <v-list-item-title>Hello</v-list-item-title>
            <v-list-item-subtitle>There how are you?</v-list-item-subtitle>
            <v-list-item-subtitle>There how are you?</v-list-item-subtitle>
          </v-list-item-content>
        </v-list-item>
      </v-list>

      <v-list-item color="white">
        <v-list-item-content>
          <v-list-item-title>Single-line item</v-list-item-title>
        </v-list-item-content>
        <v-list-item-action>
          <v-checkbox></v-checkbox>
        </v-list-item-action>
      </v-list-item>

      <div class="my-3"></div>

      <v-list-item color="white">
        <v-list-item-title>Single-line item longer text</v-list-item-title>
      </v-list-item>

      <div class="my-3"></div>

      <v-list-item color="white">
        <v-list-item-icon>
          <v-icon color="purple">mdi-bluetooth</v-icon>
        </v-list-item-icon>
        <v-list-item-content>
          <v-list-item-title>Single-line item</v-list-item-title>
        </v-list-item-content>
      </v-list-item>

      <div class="my-3"></div>

      <v-list-item color="white">
        <v-list-item-avatar color="grey">
        </v-list-item-avatar>
        <v-list-item-content>
          <v-list-item-title>Single-line item</v-list-item-title>
        </v-list-item-content>
        <v-list-item-action>
          <v-checkbox></v-checkbox>
        </v-list-item-action>
      </v-list-item>

      <div class="my-3"></div>

      <v-list-item color="white">
        <v-list-item-avatar tile size="56" color="grey">
        </v-list-item-avatar>
        <v-list-item-content>
          <v-list-item-title>Single-line item</v-list-item-title>
        </v-list-item-content>
      </v-list-item>

      <div class="my-3"></div>

      <v-list-item color="white">
        <v-list-item-avatar color="grey" height="56" width="100" horizontal></v-list-item-avatar>
        <v-list-item-content>
          <v-list-item-title>Single-line item</v-list-item-title>
        </v-list-item-content>
        <v-list-item-action>
          <v-checkbox></v-checkbox>
        </v-list-item-action>
      </v-list-item>

      <div class="my-3"></div>

      <v-list>
        <v-list-item v-for="n in 3" :key="n">
          <v-list-item-avatar color="grey"></v-list-item-avatar>
          <v-list-item-content>
            <v-list-item-title>Single-line item</v-list-item-title>
          </v-list-item-content>
          <v-list-item-action>
            <v-checkbox></v-checkbox>
          </v-list-item-action>
        </v-list-item>
      </v-list>

      <div class="my-3"></div>

      <v-list-item color="white" v-for="n in 2" :key="n">
        <v-list-item-avatar color="grey"></v-list-item-avatar>
        <v-list-item-content>
          <v-list-item-title>One-line item</v-list-item-title>
        </v-list-item-content>
        <v-list-item-action>
          <v-checkbox color="purple"></v-checkbox>
        </v-list-item-action>
      </v-list-item>

      <div class="my-3"></div>

      <v-list-item color="white" v-for="n in 2" :key="n + 10">
        <v-list-item-action>
          <v-checkbox color="purple"></v-checkbox>
        </v-list-item-action>
        <v-list-item-content>
          <v-list-item-title>One-line item</v-list-item-title>
        </v-list-item-content>
      </v-list-item>

      <div class="my-5"></div>

      <v-list-item color="white" two-line>
        <v-list-item-content>
          <v-list-item-title>Two-line item</v-list-item-title>
          <v-list-item-subtitle>Secondary text</v-list-item-subtitle>
        </v-list-item-content>
        <div>
          <v-list-item-subtitle class="overline">meta</v-list-item-subtitle>
        </div>
      </v-list-item>

      <div class="my-3"></div>

      <v-list-item color="white" two-line>
        <v-list-item-content>
          <v-list-item-subtitle class="overline">Overline</v-list-item-subtitle>
          <v-list-item-title>Two-line item</v-list-item-title>
        </v-list-item-content>
      </v-list-item>

      <div class="my-3"></div>

      <v-list-item color="white" two-line>
        <v-list-item-icon>
          <v-icon color="purple">mdi-bluetooth</v-icon>
        </v-list-item-icon>
        <v-list-item-content>
          <v-list-item-title>Two-line item</v-list-item-title>
          <v-list-item-subtitle>Secondary text</v-list-item-subtitle>
        </v-list-item-content>
      </v-list-item>

      <div class="my-3"></div>

      <v-list-item color="white" two-line>
        <v-list-item-avatar color="grey"></v-list-item-avatar>
        <v-list-item-content>
          <v-list-item-title>Two-line item</v-list-item-title>
          <v-list-item-subtitle>Secondary text</v-list-item-subtitle>
        </v-list-item-content>
        <v-list-item-action>
          <v-checkbox></v-checkbox>
        </v-list-item-action>
      </v-list-item>

      <div class="my-3"></div>

      <v-list-item color="white" two-line>
        <v-list-item-avatar tile color="grey"></v-list-item-avatar>
        <v-list-item-content>
          <v-list-item-title>Two-line item</v-list-item-title>
          <v-list-item-subtitle>Secondary text</v-list-item-subtitle>
        </v-list-item-content>
        <v-list-item-action>
          <v-checkbox></v-checkbox>
        </v-list-item-action>
      </v-list-item>

      <div class="my-3"></div>

      <v-list-item color="white" two-line>
        <v-list-item-avatar color="grey" height="56" width="100" horizontal></v-list-item-avatar>
        <v-list-item-content>
          <v-list-item-title>Two-line item</v-list-item-title>
          <v-list-item-subtitle>Secondary text</v-list-item-subtitle>
        </v-list-item-content>
        <v-list-item-action>
          <v-checkbox></v-checkbox>
        </v-list-item-action>
      </v-list-item>

      <div class="my-3"></div>

      <v-list two-line>
        <v-list-item v-for="n in 3" :key="n + 20">
          <v-list-item-avatar color="grey"></v-list-item-avatar>
          <v-list-item-content>
            <v-list-item-title>Two-line item</v-list-item-title>
            <v-list-item-subtitle>Secondary text</v-list-item-subtitle>
          </v-list-item-content>
          <v-list-item-action>
            <v-checkbox></v-checkbox>
          </v-list-item-action>
        </v-list-item>
      </v-list>

      <div class="my-5"></div>

      <v-list-item color="white" three-line>
        <v-list-item-content>
          <v-list-item-title>Three-line item</v-list-item-title>
          <v-list-item-subtitle>
            Secondary line text Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </v-list-item-subtitle>
        </v-list-item-content>
        <div>
          <v-list-item-subtitle class="overline">meta</v-list-item-subtitle>
        </div>
      </v-list-item>

      <div class="my-3"></div>

      <v-list-item color="white" three-line>
        <v-list-item-content>
          <v-list-item-subtitle class="overline">Overline</v-list-item-subtitle>
          <v-list-item-title>Three-line item</v-list-item-title>
          <v-list-item-subtitle>
            Secondary line text Lorem ipsum dolor sit amet
          </v-list-item-subtitle>
        </v-list-item-content>
      </v-list-item>

      <div class="my-3"></div>

      <v-list-item color="white" three-line>
        <v-list-item-icon>
          <v-icon color="purple">mdi-bluetooth</v-icon>
        </v-list-item-icon>
        <v-list-item-content>
          <v-list-item-title>Three-line item</v-list-item-title>
          <v-list-item-subtitle>
            Secondary line text Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </v-list-item-subtitle>
        </v-list-item-content>
      </v-list-item>

      <div class="my-3"></div>

      <v-list-item color="white" three-line>
        <v-list-item-avatar color="grey"></v-list-item-avatar>
        <v-list-item-content>
          <v-list-item-title>Three-line item</v-list-item-title>
          <v-list-item-subtitle>
            Secondary line text Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </v-list-item-subtitle>
        </v-list-item-content>
        <v-list-item-action>
          <v-checkbox></v-checkbox>
        </v-list-item-action>
      </v-list-item>

      <div class="my-3"></div>

      <v-list-item color="white" three-line>
        <v-list-item-avatar color="grey" tile></v-list-item-avatar>
        <v-list-item-content>
          <v-list-item-title>Three-line item</v-list-item-title>
          <v-list-item-subtitle>
            Secondary line text Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </v-list-item-subtitle>
        </v-list-item-content>
        <v-list-item-action>
          <v-checkbox></v-checkbox>
        </v-list-item-action>
      </v-list-item>

      <div class="my-3"></div>

      <v-list-item color="white" three-line>
        <v-list-item-avatar color="grey" horizontal width="100" height="56"></v-list-item-avatar>
        <v-list-item-content>
          <v-list-item-title>Three-line item</v-list-item-title>
          <v-list-item-subtitle>
              Secondary line text lorem ipsum dapibus, neque
          </v-list-item-subtitle>
        </v-list-item-content>
        <v-list-item-action>
          <v-checkbox></v-checkbox>
        </v-list-item-action>
      </v-list-item>

      <div class="my-3"></div>

      <v-list three-line>
        <v-list-item v-for="n in 3" :key="n + 40">
          <v-list-item-avatar color="grey" horizontal width="100" height="56"></v-list-item-avatar>
          <v-list-item-content>
            <v-list-item-title>Three-line item</v-list-item-title>
            <v-list-item-subtitle>
              Secondary line text lorem ipsum dapibus, nequeg
            </v-list-item-subtitle>
          </v-list-item-content>
          <v-list-item-action>
            <v-checkbox></v-checkbox>
          </v-list-item-action>
        </v-list-item>
      </v-list>
    </div>
    <v-flex xs12 sm2 offset-sm3>
      <v-card>
        <v-toolbar color="teal" dark>
          <v-toolbar-side-icon></v-toolbar-side-icon>

          <v-toolbar-title>Topics</v-toolbar-title>

          <v-spacer></v-spacer>

          <v-btn icon>
            <v-icon>more_vert</v-icon>
          </v-btn>
        </v-toolbar>

        <v-list dense>
          <v-list-group
            v-for="item in items"
            :key="item.title"
            v-model="item.active"
            :prepend-icon="item.action"
            no-action
            @click="test"
          >
            <template v-slot:activator>
              <v-list-item-content>
                <v-list-item-title>{{ item.title }}</v-list-item-title>
              </v-list-item-content>
            </template>

            <v-list-item
              v-for="subItem in item.items"
              :key="subItem.title"
              @click=""
            >
              <v-list-item-content>
                <v-list-item-title>{{ subItem.title }}</v-list-item-title>
              </v-list-item-content>

              <v-list-item-action>
                <v-icon>{{ subItem.action }}</v-icon>
              </v-list-item-action>
            </v-list-item>
          </v-list-group>

          <v-list-group no-action>
            <template #activator>
              <v-list-item-content>
                <v-list-item-title>I'm a title that is way too long and should truncate</v-list-item-title>
                <v-list-item-subtitle>I'm a sub titleg</v-list-item-subtitle>
              </v-list-item-content>
            </template>
            <v-list-group sub-group no-action>
              <template #activator>
                <v-list-item>
                  <v-list-item-content>
                    <v-list-item-title>
                      A nested group
                    </v-list-item-title>
                  </v-list-item-content>
                </v-list-item>
              </template>

              <v-list-item @click="">
                <v-list-item-content>
                  <v-list-item-title>A nested item</v-list-item-title>
                </v-list-item-content>
              </v-list-item>
            </v-list-group>
          </v-list-group>
        </v-list>
      </v-card>
    </v-flex>
  </v-layout>
</template>

<script>
  export default {
    data () {
      return {
        items: [
          {
            action: 'local_activity',
            title: 'Attractionsg',
            items: [
              { title: 'List Item' }
            ]
          },
          {
            action: 'restaurant',
            title: 'Dining',
            active: true,
            items: [
              { title: 'Breakfast & brunch' },
              { title: 'New American' },
              { title: 'Sushi' }
            ]
          },
          {
            action: 'school',
            title: 'Education',
            items: [
              { title: 'List Item' }
            ]
          },
          {
            action: 'directions_run',
            title: 'Family',
            items: [
              { title: 'List Item' }
            ]
          },
          {
            action: 'healing',
            title: 'Health',
            items: [
              { title: 'List Item' }
            ]
          },
          {
            action: 'content_cut',
            title: 'Office',
            items: [
              { title: 'List Item' }
            ]
          },
          {
            action: 'local_offer',
            title: 'Promotions',
            items: [
              { title: 'List Item' }
            ]
          }
        ]
      }
    },

    methods: {
      test () {
        console.log('here')
      }
    }
  }
</script>

```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/docs/src/examples/list-item-groups/selectionControls.vue [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            :key="`divider-${i}`"
+          ></v-divider>
+
+          <v-list-item
+            v-else
+            :key="`item-${i}`"
+            :value="item"
+            active-class="deep-purple--text text--accent-4"
+          >
+            <template #default="{ active, toggle }">
```
Comment (jacekkarczmarczyk) [inline packages/docs/src/examples/list-item-groups/selectionControls.vue]: https://github.com/vuetifyjs/vuetify/pull/6502#issuecomment-464061772

File: packages/vuetify/src/components/VList/VList.sass [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+// Theme
+.v-list.theme--light
+  +list-theme($material-light)
+.v-list.theme--dark
+  +list-theme($material-dark)
+
+// Block
+.v-list
+  padding: 8px 0
```
Comment (sh7dm) [inline packages/vuetify/src/components/VList/VList.sass]: move paddings to variables?

File: packages/vuetify/src/components/VList/VListGroup.sass [hunk 3]
```diff
@@ -0,0 +1,66 @@
+@import '../../styles/styles.sass'
+@import './_variables.scss'
+@import './_mixins.sass'
+
+// Element
+.v-list-group
+  .v-list-item__icon.v-list-group__header__append-icon
+    min-width: 48px
```
Comment (sh7dm) [inline packages/vuetify/src/components/VList/VListGroup.sass]: var it!

File: packages/vuetify/src/components/VList/VListGroup.ts [hunk 4]
```diff
@@ -56,54 +60,36 @@ export default mixins<options &
...
+        'v-list-group--active': this.isActive,
+        'v-list-group--disabled': this.disabled,
+        'v-list-group--no-action': this.noAction,
+        'v-list-group--sub-group': this.subGroup
       }
     }
   },
 
   watch: {
     isActive (val) {
```
Comment (sh7dm) [inline packages/vuetify/src/components/VList/VListGroup.ts]: `val: boolean`

File: packages/vuetify/src/components/VList/VListGroup.ts [hunk 5]
```diff
@@ -133,21 +117,30 @@ export default mixins<options &
...
+        staticClass: 'v-list-group__header__append-icon'
       }, [
         this.$slots.appendIcon || this.genIcon(icon)
       ])
     },
-    genGroup () {
-      return this.$createElement('div', {
-        staticClass: 'v-list__group__header',
-        class: this.headerClasses,
+    genHeader () {
```
Comment (sh7dm) [inline packages/vuetify/src/components/VList/VListGroup.ts]: `: VNode`

File: packages/vuetify/src/components/VList/VListGroup.ts [hunk 6]
```diff
@@ -156,14 +149,14 @@ export default mixins<options &
     },
     genItems () {
```
Comment (sh7dm) [inline packages/vuetify/src/components/VList/VListGroup.ts]: `: VNode`

File: packages/vuetify/src/components/VList/VListItem.sass [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+  font-size: map-deep-get($headings, 'caption', 'size')
+
+.v-list-item__avatar
+  align-self: center
+  justify-content: flex-start
+  margin-bottom: $list-item-avatar-margin-y
+  margin-top: $list-item-avatar-margin-y
+
+  &.v-list-item__avatar--horizontal
+    margin-bottom: 8px
```
Comment (sh7dm) [inline packages/vuetify/src/components/VList/VListItem.sass]: Margins should be vars

File: packages/vuetify/src/components/VList/VListItemAction.ts [hunk 8]
```diff
@@ -3,16 +3,16 @@ import Vue, { VNode } from 'vue'
...
 /* @vue/component */
 export default Vue.extend({
-  name: 'v-list-tile-action',
+  name: 'v-list-item-action',
 
   functional: true,
 
-  render (h, { data, children = [] }): VNode {
-    data.staticClass = data.staticClass ? `v-list__tile__action ${data.staticClass}` : 'v-list__tile__action'
+  render (h, { data, children = [], props }): VNode {
```
Comment (sh7dm) [inline packages/vuetify/src/components/VList/VListItemAction.ts]: Are `props` required here?

File: packages/vuetify/src/components/VList/VListItemIcon.ts [hunk 9]
```diff
@@ -0,0 +1,15 @@
+// Types
+import Vue, { VNode } from 'vue'
+
+/* @vue/component */
+export default Vue.extend({
+  name: 'v-list-item-icon',
+
+  functional: true,
+
+  render (h, { data, children, props }): VNode {
```
Comment (sh7dm) [inline packages/vuetify/src/components/VList/VListItemIcon.ts]: Looks like `props` here aren't used

Comment (sh7dm) [reply packages/vuetify/src/components/VList/VListItemIcon.ts]: VSCode thinks so
изображение

**Comment by sh7dm** [comment]
@johnleider please review

**Pull request merged**